### DISCUSSION TITLE
Karokta (Zaralek Rares) Question

### DISCUSSION THREAD

**Discussion by blehmonster** [SavedInstances/SavedInstances#821] [Q&A] [answered]
Hi all- I use SavedInstances to track daily/weekly kills for mount farming.  I have been farming Karokta in Zaralek Caverns for months.  On some days I can kill him 5–10 times across characters.  Would love to consolidate this information into SavedInstances so I have one HUD for everything.  These rares follow an odd and unique(?) rotation within WoW.  

The rotation is detailed here: https://docs.google.com/spreadsheets/d/e/2PACX-1vRrnjYASgBC_8UD9PKE0AORnigDgQ9K2QJZTXQGifBxUH-YBfafxzVP4RKqtj1JTACbSjju367eSSod/pubhtml

Does anyone have a suggestion on how I could track this mob across characters using SavedInstances?  Thanks!

**Comment by LiangYuxuan** [comment, marked as answer]
You can add the rare tracking quest in SI config > Quest Progress > User.

**Reply by ereidy1** [reply]
Can you explain the "Full Objective" and "Progress Persists" options?

**Reply by LiangYuxuan** [reply]
These options is designed for normal quests.

"Progress Persists" is for weekly quests that don't reset themselves on weekly reset but remains the same progress, like Aiding the Accord, A Worthy Ally, Patterns Within Patterns.
"Full Objective" is for quests that more than one objective is needed to track, like Aiding the Accord, has 3000 reps and one special objective changes weekly which are all needed to track.

For rare tracking quests, there're no objective progress and ofc no progress to persists, only completed or not.

**Reply by ereidy1** [reply]
great info, thanks!

**Comment by blehmonster** [comment]
This is perfect, thanks.

**Comment by blehmonster** [comment]
@LiangYuxuan 
Does this trigger "Show when saved"? I have killed this rare on several characters this week, but they don't show up under the SavedInstances display unless I hold Alt to show all. I checked under Characters in the menu, and they're set to "Show when saved".
